### DISCUSSION TITLE
See new wiki entry: Crouton Quick Start

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#5021] [Show and tell]
[Crouton Quick Start](https://github.com/dnschneid/crouton/wiki/Crouton-Quick-Start) Feedback not required but is welcome.

**Comment by dp0100** [comment]
Thanks for posting and worked for me! I have an issue with the browser, getting a fatal error that shuts down the session. Checked network and wifi is not enabled, nor am I able to do anything, all grayed out. I've tried to load the intel drivers and place in the /lib/firmware folder but still nothing?  I'm a noob with linux so what is the best way to get connected. Appreciate any help

**Reply by CroutonIsFun** [reply]
You're trying to run a browser inside your chroot? If you own a Chromebook, you don't need any drivers. Does chrome itself work? Internet access inside crouton is provided by crouton itself.

**Reply by dp0100** [reply]
Thanks and yes, was just checking everything out in the new chroot and tried to launch the browser but no luck, was kicked out of the session.
Yes, ChromeOS works fine, , no issues there and also read that the internet connection just works so must be something in the build. There are a bunch of warnings and errors so thinking I'll just create a new chroot to see if that fixes it. 

Appreciate the response and will post back

**Reply by CroutonIsFun** [reply]
Cool. If you wish to run Firefox, another option is to install the Android app. It will look good on a Chromebook screen. If you have no browsers installed in your chroot, then the default browser is _croutonurlhandler_, provided that you have installed the crouton extension.

**Reply by dp0100** [reply]
Thanks again and yes, was able to install Firefox and good to go!  Anyway to get audio working?  I've tried the remove/install alsa-base and crouton -u -t audio,core -n chroot and no luck.

**Reply by CroutonIsFun** [reply]
@dp0100 

No, you're taking the wrong approach. Without the cras from autotools, your chroot is the same as a Linux install, wherein the Linux kernel doesn't recognize the hardware.

You will need to purchase an inexpensive USB audio adapter, like the one Sabrent sells. (Check Amazon, for example). Look at the following how-to:

[How to get sound in crouton](https://github.com/dnschneid/crouton/discussions/4969)

**Reply by dp0100** [reply]
Thank you and I'll check it out
